index.markdown

```diff
@@ -1,10 +1,75 @@
 ---
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
 layout: default
 ---
 
-# Big Test Energy
+# UWB Hacks 2020 Documentation
+
+![logo](https://cdn.discordapp.com/attachments/652802883531571210/668393411564404736/websiteDesign1.png)
+Temporary logo placeholder using an awesome design created by Erica
+
+Welcome to the documentation page for UWB ACM's annual hackathon! Check out 
+some of the guides below to prepare for the big day, find inspiration
+for potential projects, or get help if you are feeling lost or stuck. 
+
+
+## Schedule
+  
+TODO: This is mostly pulled from last year's schedule
+
+| Time     | Event                           |
+|---------:|:--------------------------------|
+|  9:15 AM | Brekkie time                    |
+| 10:00 AM | Opening Ceremony                |
+| 10:30 AM | Team-Building Mixer             |
+| 11:15 AM | Cloud Computing Workshop #1     |
+| 12:00 PM | Free Lunch Provided             |
+| 12:45 PM | Cloud Computing Workshop #2     |
+|  2:45 PM | Greyhats Workshop?              |
+|  6:00 PM | Free Dinner Provided            |
+|  9:15 PM | Project Presentations           |
+| 10:00 PM | Awards & Closing Ceremony       |
+
+## Submitting Your Project
+
+1. Submit your project to devpost? 
+2. Maybe general outline of steps here on index page, then link to a page with 
+more in-depth instructions under logistics section of useful info
+
+
+## Useful Info
+
+#### Getting Started
+- [Preparing for the Hackathon]()
+- [Project Ideas]()
+- [What is _the cloud_?]()
+- [Configuring Your Cloud Environment]()
+- [Collaboration Tools]()
 
-This will be the "welcome page" for the docs website.
+#### Logistics
+ - [Getting Help]() 
+ - [Forming a Team]()
+ - [Project Submission]()
+ - [Hackathon Rules]()
+ - [Prize Categories & Criteria]()
+ 
+#### Cloud Services
+- [Data Storage]()
+  - [AWS: Simple Storage Services (S3)]()
+  - [Azure Storage]()
+  - [Google Cloud Storage]()
+- [Database Services]()
+  - [AWS: DynamoDB]()
+  - [Azure: Cosmos DB]()
+  - [Google Cloud Firestore]()
+- [Managed Notification Services]()
+  - [AWS: Simple Notification Service (SNS)]()
+  - [Azure Notification Hubs]()
+  - [Google Cloud Pub/Sub]()
+- [Serverless Compute]()
+  - [AWS Lambda]()
+  - [Azure Serverless]()
+  - [Google Cloud Functions]()
+- [Web App Deployment]()
+  - [AWS Elastic Beanstalk]()
+  - [Azure Web App Services]()
+  - [Google Cloud App Engine]()
```

Comment:
This is a fine layout IMO. I envision each of the Services sections to have a single markdown page, but H1 & H2 headings should allow for internal heading links on the page. We can test that theory later in the process.

One more thing -- we should have a docs page that links out to other common, important, or interesting services (ML solutions, CloudWatch for logging, IAM for secrets/auth/permissions, etc); this hasn't been delegated out yet but is something we should put together.

Reply:
Yeah, it looks like we should be able to add anchors to the headings on each page which will _hopefully_ allow us to link directly to them. 

I can't decide if it'd be better to comment out the cloud services table of contents for now while we're working on the content for those sections or write up a generic "more info coming soon!" page for each type of service and link to that instead - let me know if you have any thoughts!